ios/brave-ios/Sources/BraveWallet/Settings/CustomNetworkDetailsView.swift

```diff
@@ -424,12 +458,31 @@ struct NetworkDetailsView: View {
           }
         )
     )
+    .background(
+      Color.clear
+        .alert(
+          isPresented: $isPresentingEditConfirmation,
+          content: {
+            Alert(
+              title: Text(Strings.Wallet.editExistingNetworkAlertMsg),
+              primaryButton: .cancel(),
```

Comment:
Why is the `primaryButton` cancel?

Reply:
updated in [f7a92a3](https://github.com/brave/brave-core/pull/24225/commits/f7a92a30662d6d519ace0378c07878b488b046c9)